src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/viewtakes/view/ViewTakesStyles.kt

```diff
@@ -52,8 +63,10 @@ class ViewTakesStyles : Stylesheet() {
             backgroundColor += AppTheme.colors.cardBackground.deriveColor(0.0, 1.0, 1.0, 0.8)
             borderRadius += box(10.px)
             backgroundRadius += box(10.px)
-            maxHeight = 100.px
-            maxWidth = 250.px
+            minHeight = 200.px
+            minWidth = 348.px
```

Comment:
...what an oddly specific arbitrary number o.O

Reply:
from the design specs

Reply:
what are you looking at though? all of the take cards I've seen are 392w 208h?

Reply:
resolved: unaware of spec change